snapcraft/integrations/__init__.py

```diff
@@ -45,6 +45,10 @@ def enable_ci(ci_system):
     module = importlib.import_module(
         'snapcraft.integrations.{}'.format(ci_system))
 
+    if refresh_only:
+        module.refresh()
+        return
```

Comment:
This is a pet peeve so you can ignore me if you want, but maybe split into some methods or wrap what follows in an `else` clause? I just don't like empty `return` statements